products/jetty.md

```diff
@@ -0,0 +1,107 @@
+title: Eclipse Jetty
+category: server-app
+tags: eclipse web-server
+iconSlug: eclipsejetty
+permalink: /jetty
+releasePolicyLink: https://eclipse.dev/jetty/download.php
+changelogTemplate: "https://github.com/jetty/jetty.project/releases/tag/jetty-{{'__LATEST__'}}"
+releaseDateColumn: true
+eolColumn: "Community Support"
+extendedSupportColumn: "Extended Support"
+
+identifiers:
+-   purl: pkg:maven/org.eclipse.jetty/jetty-server
+
+auto:
+-   maven: org.eclipse.jetty/jetty-server
+
+releases:
+-   releaseCycle: "12"
+    releaseDate: 2023-08-07
+    eol: false
+    extendedSupport: true
+    minJavaVersion: 17
+    latest: "12.0.3"
+    latestReleaseDate: 2023-10-30
+
+-   releaseCycle: "11"
+    releaseDate: 2020-12-07
+    eol: false
+    extendedSupport: true
+    minJavaVersion: 11
+    latest: "11.0.18"
+    latestReleaseDate: 2023-10-30
+
+-   releaseCycle: "10"
+    releaseDate: 2020-12-07
+    eol: false
+    extendedSupport: true
+    minJavaVersion: 11
+    latest: "10.0.18"
+    latestReleaseDate: 2023-10-30
+
+-   releaseCycle: "9.4"
+    releaseDate: 2016-12-07
+    eol: 2022-06-01
+    extendedSupport: true
+    minJavaVersion: 8
+    latest: "9.4.53.v20231009"
+    latestReleaseDate: 2023-10-10
+
+-   releaseCycle: "9.3"
+    releaseDate: 2015-06-01
+    eol: 2021-10-01
```

Comment:
Shouldn't we use the dates indicated on https://eclipse.dev/jetty/download.php as EOL dates, even if they give no month nor dates (we can use 12-31 as the month / day) ?